utils/sites.js

```diff
@@ -330,6 +330,12 @@ const updateParticipantData = async (req, res, authObj) => {
             delete flatUpdateObj[fieldMapping.participantDeceasedTimestamp];
         }
 
+        // Handle destroyed data. If destroyData = yes then participation status = destroyData
+        // https://github.com/episphere/connect/issues/1117#issuecomment-2394398584
```

Comment:
Link to issue comments can be in PR description. No need to be in code.

Reply:
Solved